src/Functions/createLink.spec.js

```diff
@@ -31,9 +31,9 @@ describe('createLink', () => {
   test('It can shorten very long URLs', async () => {
     const url =
       'https://www.dosomething.org/us/articles/13-ways-the-sports-world-' +
-      'is-stepping-up-during-the-coronavirus-pandemic?utm_source=content' +
-      '_campaign&utm_medium=sms&utm_campaign=sms_weekly_2020_05_11&user_' +
-      'id=55be62d4469c64182b91992b&broadcast_id=6v1RJUUmrWN2Ode0EW6lH';
+      'is-stepping-up-during-the-coronavirus-pandemic?broadcast_id=6v1RJ' +
+      'UUmrWN2Ode0EW6lH&user_id=55be62d4469c64182b91992b&utm_campaign=sm' +
+      's_weekly_2020_05_11&utm_medium=sms&utm_source=content_campaign';
```

Comment:
Why the URL parameter change order?

Reply:
Our URL normalization logic now sorts query parameters, and I didn't want this test to be reliant on that behavior (as it's tested separately below).